src/styles/themes.ts

```diff
@@ -2,6 +2,8 @@ import { DefaultTheme } from 'styled-components'
 
 export type ThemeType = 'light' | 'dark'
 
+export const oppositeThemeOf = (theme) => (theme === darkTheme ? lightTheme : darkTheme)
```

Comment:
This causes an editor error linked to type: `any`.

<img width="496" alt="image" src="https://user-images.githubusercontent.com/6715943/171117885-40abee21-1d8b-463f-bdfc-cb93f541fb6e.png">


Also, this is not used anywhere, so what should we do with it? :)

Reply:
Actually, seems like your local typescript compiler isn't screaming at you? Mine is upset :D 

<img width="622" alt="image" src="https://user-images.githubusercontent.com/6715943/171140438-7d0056e2-8d75-449e-8492-b696613b2ef0.png">